.travis.yml

```diff
@@ -24,23 +24,117 @@ matrix:
+      dist: trusty
+      env: PLATFORMS=debian_puppet6
+      rvm: 2.5.3
+      before_script:
+      - bundle exec rake 'litmus:provision_list[travis_debian]'
+      - bundle exec rake 'litmus:install_agent[puppet6]'
+      - bundle exec rake litmus:install_module
+      script:
+      - bundle exec rake litmus:acceptance:parallel
+      services: docker
+      sudo: required
+      stage: acceptance
+    -
+      bundler_args:
+      dist: trusty
+      env: PLATFORMS=ubuntu_puppet5
+      rvm: 2.5.3
+      before_script:
+      - bundle exec rake 'litmus:provision_list[travis_ubuntu]'
+      - bundle exec rake 'litmus:install_agent[puppet5]'
+      - bundle exec rake litmus:install_module
+      script:
+      - bundle exec rake litmus:acceptance:parallel
+      services: docker
+      sudo: required
+      stage: acceptance
+    -
+      bundler_args:
+      dist: trusty
+      env: PLATFORMS=ubuntu_puppet6
+      rvm: 2.5.3
+      before_script:
+      - bundle exec rake 'litmus:provision_list[travis_ubuntu]'
+      - bundle exec rake 'litmus:install_agent[puppet6]'
+      - bundle exec rake litmus:install_module
+      script:
+      - bundle exec rake litmus:acceptance:parallel
+      services: docker
+      sudo: required
       stage: acceptance
+    -
+      bundler_args:
+      dist: trusty
+      env: PLATFORMS=el6_puppet5
+      rvm: 2.5.3
+      before_script:
+      - bundle exec rake 'litmus:provision_list[travis_el6]'
+      - bundle exec rake 'litmus:install_agent[puppet5]'
+      - bundle exec rake litmus:install_module
+      script:
+      - bundle exec rake litmus:acceptance:parallel
+      services: docker
+      sudo: required
+      stage: acceptance
+    -
+      bundler_args:
+      dist: trusty
+      env: PLATFORMS=el6_puppet6
+      rvm: 2.5.3
+      before_script:
+      - bundle exec rake 'litmus:provision_list[travis_el6]'
+      - bundle exec rake 'litmus:install_agent'
+      - bundle exec rake litmus:install_module
+      script:
+      - bundle exec rake litmus:acceptance:parallel
+      services: docker
       sudo: required
+      stage: acceptance
     -
-      bundler_args: 
+      bundler_args:
       dist: trusty
-      env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_PUPPET_COLLECTION=puppet6 BEAKER_set=docker/ubuntu-14.04 BEAKER_TESTMODE=apply
+      env: PLATFORMS=el7_puppet5
       rvm: 2.5.3
-      script: bundle exec rake beaker
+      before_script:
+      - bundle exec rake 'litmus:provision_list[travis_el7]'
+      - bundle exec rake 'litmus:install_agent[puppet5]'
+      - bundle exec rake litmus:install_module
+      script:
+      - bundle exec rake litmus:acceptance:parallel
       services: docker
+      sudo: required
       stage: acceptance
+    -
+      bundler_args:
+      dist: trusty
+      env: PLATFORMS=el7_puppet6
+      rvm: 2.5.3
+      before_script:
+      - bundle exec rake 'litmus:provision_list[travis_el7]'
+      - bundle exec rake 'litmus:install_agent'
```

Comment:
`[puppet6]`

Reply:
not needed, by default it will do puppet6. i dont think i can take another travis run